backend/src/controllers/employee.controller.js

```
@@ -0,0 +1,253 @@
+      .status(500)
+      .json({ massage: "internal sever error", success: false, data: null });
+    console.log(error.message);
+  }
+};
+
+const getEmployeeById = async () => {
+  const employeeId = req.params.id;
+  const userId = req.userId; //via middleware
+
+  try {
+    const requestedUser = await User.findById(userId);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)